site/src/components/Workspace/Workspace.stories.tsx

```diff
@@ -24,21 +24,37 @@ export default {
   ],
 }
 
+const MockedAppearance = {
+  config: Mocks.MockAppearance,
+  preview: false,
+  setPreview: () => null,
+  save: () => null,
+}
+
 const Template: Story<WorkspaceProps> = (args) => (
-  <ProxyContext.Provider
+  <DashboardProviderContext.Provider
     value={{
-      proxyLatencies: MockProxyLatencies,
-      proxy: getPreferredProxy([], undefined),
-      proxies: [],
-      isLoading: false,
-      isFetched: true,
-      setProxy: () => {
-        return
-      },
+      buildInfo: Mocks.MockBuildInfo,
+      entitlements: Mocks.MockEntitlementsWithScheduling,
+      experiments: Mocks.MockExperiments,
+      appearance: MockedAppearance,
     }}
   >
-    <Workspace {...args} />
-  </ProxyContext.Provider>
+    <ProxyContext.Provider
```

Comment:
Not sure if I'm correct, but maybe, this ProxyContext is already inside of the DashboardProvider so we don't need that here too?

Reply:
I can check!

Reply:
Sadly both are needed